src/custom-code-helpers/console-output/templates/ConsoleOutputDisableExpressionTemplate.ts

```diff
@@ -3,36 +3,20 @@
  */
 export function ConsoleOutputDisableExpressionTemplate (): string {
     return `
-        const {consoleLogDisableFunctionName} = {callControllerFunctionName}(this, function () {
-            const func = function () {};
-            
+        const {consoleLogDisableFunctionName} = {callControllerFunctionName}(this, function () {            
```

Comment:
I know it's a trifle but you have some excessive spaces.
I didn't look for them on purpose, just randomly found and shared with you

![image](https://user-images.githubusercontent.com/12462454/89378600-d4e37300-d6fc-11ea-8374-a46dafdc972e.png)

Reply:
Thank you. Will fix it